src/inabox/inabox-iframe-messaging-client.js

```diff
@@ -42,7 +43,13 @@ export function installIframeMessagingClient(win) {
  */
 function createIframeMessagingClient(win) {
   const iframeClient = new IframeMessagingClient(win);
-  iframeClient.setSentinel(getRandom(win));
+  //  Try read sentinel from window first.
+  const dataObject = tryParseJson(win.name);
+  let sentinel = null;
+  if (dataObject && dataObject['_context']) {
+    sentinel = dataObject['_context']['sentinel'];
```

Comment:
@alanorozco hi Alan, to raise this in your radar.
how does this affect your work? can this get in first and you clean it up later?